lib/rspec_profiling/collectors/statsd.rb

```diff
@@ -88,9 +87,33 @@ def formatFile(path)
         path.gsub('/', '.').gsub(/\.rb$|^\.[^.]*\./, '')
       end
 
+      def buildStamp(desc, lineNumber)
```

Comment:
Ruby methods should be lowercase underscore: `build_stamp` and `format_readable_short`

Reply:
parameter names as well.